resources/migrations/000_migrations.yaml

```diff
@@ -16096,6 +16096,59 @@ databaseChangeLog:
               column:
                 name: action_id
 
+  - changeSet:
```

Comment:
Is there any benefit to having some timestamp column here, sth like `last_synced_at`?

Reply:
I can't think of any reason to need that. The data here is just a cache and invalidation shouldn't be based on time. I also feel like we can introduce a timestamp if we need one later.